frontend/app/dashboard/src/views/dashboard/webshop/edit/products/EditProductView.vue

```diff
@@ -9,6 +9,21 @@
 
         <STErrorsDefault :error-box="errors.errorBox" />
 
+        <p v-if="!patchedProduct.uitpasEvent && atLeastOneUitpasSocialTariff && uitpasFeature && organization.meta.uitpasClientCredentialsStatus === UitpasClientCredentialsStatus.Ok" class="warning-box with-button selectable" @click="openUitpasEventSearch()">
+            {{ $t('Er werd géén UiTPAS-evemenent gekoppeld, je krijgt géén automatische terugbetaling van je UiTPAS-regio.') }}
+
+            <button class="button text" type="button">
+                {{ $t('UiTPAS-evenement koppelen') }}
+            </button>
+        </p>
```

Comment:
Als je de onClick toevoegt op de warning-box, moet die eigenlijk button worden
```suggestion
        <button type="button" v-if="!patchedProduct.uitpasEvent && atLeastOneUitpasSocialTariff && uitpasFeature && organization.meta.uitpasClientCredentialsStatus === UitpasClientCredentialsStatus.Ok" class="warning-box with-button selectable" @click="openUitpasEventSearch()">
            {{ $t('Er werd géén UiTPAS-evemenent gekoppeld, je krijgt géén automatische terugbetaling van je UiTPAS-regio.') }}

            <span class="button text">
                {{ $t('UiTPAS-evenement koppelen') }}
            </span>
        </button>
```